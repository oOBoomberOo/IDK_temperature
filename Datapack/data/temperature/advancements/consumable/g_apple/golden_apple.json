{
	"criteria": {
		"golden_apple": {
			"trigger": "minecraft:consume_item",
			"conditions": {
				"item": {
					"item": "minecraft:golden_apple"
				}
			}
		}
	},
	"parent": "temperature:consumable/root",
	"rewards": {
		"function": "temperature:temp_calculation/consumable/g_apple/golden_apple"
	}
}