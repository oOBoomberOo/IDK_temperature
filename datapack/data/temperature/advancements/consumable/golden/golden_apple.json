{
	"criteria": {
		"golden_apple": {
			"trigger": "minecraft:consume_item",
			"conditions": {
				"item": {
					"item": "minecraft:golden_apple"
				}
			}
		}
	},
	"parent": "temperature:consumable/root",
	"rewards": {
		"function": "temperature:temp_calculation/consumable/golden/golden_apple"
	}
}